banyan/tests/build_check.rs

```diff
@@ -133,6 +136,22 @@ fn filtered_chunked_no_holes(t: TestTree, range: Range<u64>) -> anyhow::Result<b
     Ok(max_offset == (xs.len() as u64))
 }
 
+/// checks that stream_filtered_chunked returns the same elements as filtering each element manually
```

Comment:
these two tests don't do manual filtering, no?

Reply:
No, there are 2 tests that filtering a tree is equivalent to filtering the elements. And this test is to make sure that reverse order produces the same elements as forward order.

Probably redundant, but also no harm done.